ttnn/cpp/ttnn/operations/eltwise/binary_backward/device/binary_backward_op.cpp

```diff
@@ -272,6 +272,17 @@ std::vector<ttnn::Tensor> _squared_difference_bw(
 }
 
 
+std::vector<Tensor> _remainder_bw(
+    const Tensor& grad, const Tensor& input, const Tensor& other, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    grad_tensor.emplace_back(grad);
+    Tensor result_div = div(input, other, true, "floor");
+    Tensor grad_b = ttnn::multiply(ttnn::neg(grad), result_div, std::nullopt, output_mem_config);
```

Comment:
Would you sanity check this as well?   
```
 Tensor grad_b = ttnn::multiply(ttnn::neg(result_div), grad, std::nullopt, output_mem_config);
```

Reply:
@rtawfik01, is there a clever trick that we can do here to get around the the DIV precision issue?